.vscode/launch.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Looks like a little cruft made this into your PR! Mind omitting your VS Code config and DS_Store from version control?

Reply:
Got it! Uh but I realized I actually don't know how to remove DS_Store -- it doesn't show up in my local repository now... a6f3271c0ac4189972819d9eda4570ba921fbd9e